feature.rb

```diff
@@ -0,0 +1,31 @@
+require './lib/order.rb'
+
+new_menu = Menu.new
+puts new_menu.menu
+puts 'What would you like to order?'
+
+menu_item = gets.chomp.to_sym
+
+puts 'How many of those would you like?'
+item_number = gets.chomp.to_i
+
+order = Order.new
+order.start_order(menu_item, item_number)
+puts order.current_order
+
+while true
```

Comment:
Style/InfiniteLoop: Use `Kernel#loop` for infinite loops. (https://github.com/bbatsov/ruby-style-guide#infinite-loop)